web_console/components/Header.jsx

```diff
@@ -152,6 +153,22 @@ export default function Header() {
     return () => document.removeEventListener('scroll', scrollHandler);
   }, [scrollTop]);
 
+  const displayFederationFilter = [
+    '/job',
+    '/ticket',
+    '/raw_data',
+  ].some(el => el === route)
+  const { data: fedData } = useSWR('federations', fetcher)
+  const federations = fedData ? fedData.data : null
+  const [currFederation, setCurrFederation] = useState(-1)
+  useEffect(() => {
+    setTimeout(() => setCurrFederation(parseInt(localStorage.getItem('federationID')) || -1))
```

Comment:
why use `setTimeout` inside `useEffect`?

Reply:
cause we use ssr, localStorage is not available while rendering. Here needs to get storage after mount and fire rerender.

Reply:
what's the problem? useEffect's callback would only be executed after component mounted

Reply:
i didn't use `setTimeout` beginning, but `setCurrFederation` did not fire the rerender

Reply:
i tried again, it works as expected. i'll modify here